saleor/webhook/tests/test_webhook_serializers.py

```diff
@@ -73,52 +74,46 @@ def test_serialize_product_attributes(
     ]
 
 
+ATTRIBUTES = sentinel.ATTRIBUTES
+
+
+@patch(
+    "saleor.webhook.serializers.serialize_product_or_variant_attributes",
+    new=Mock(return_value=ATTRIBUTES),
+)
+@pytest.mark.parametrize("taxes_included", [True, False])
+# TODO: split this test by test function
```

Comment:
Please split this test instead of leaving `TODO`

Reply:
Resolved in recent commit.